### PR TITLE
Create terraform graph dir for production DNS

### PR DESCRIPTION
**Pull request by meatballhat**
## What is the problem that this PR is trying to fix?

The documentation for which IP addresses we use in GCE is outdated, reporting a much larger range than we now use with NAT stuff in place.

## What approach did you choose and why?

Create a single DNS record that contains A records for all of the IP addresses assigned to production GCE NAT hosts, with the intent of listing this single record in the documentation.

## How can you test this?

- [x] `make plan` is happy

### CODE REVIEW

File: dns-production-0/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+ENV := dns
+ENV_SHORT := production
+INDEX := 0
+
+include $(shell git rev-parse --show-toplevel)/terraform-common.mk
+
+PROD_TF_VERSION := v0.11.2
```
Comment (Lyoness) [inline dns-production-0/Makefile]: We've pinned terraform version to v0.9.11 because of the vpshere provider changes in the 0.10+ version change.

Comment (Lyoness) [reply dns-production-0/Makefile]: err. pinned for macstadium

Comment (Lyoness) [reply dns-production-0/Makefile]: Hardcoding the version might conflict with (bin/ensure-terraform)[https://github.com/travis-ci/terraform-config/blob/master/bin/ensure-terraform#L13] arch check ?

Comment (meatballhat) [reply dns-production-0/Makefile]: @Lyoness this version pin will only apply to this graph/dir :+1:

**Pull request merged**